apps/nextra/pages/en/build/sdks/swift-sdk/quickstart.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: "Swift SDK Quickstart"
+---
+
+import { Steps } from 'nextra/components'
+
+# Swift SDK Quickstart
+
+This guide will walk you through the process of setting up **AptosKit**, and
+fetching data on the Aptos blockchain.
+
+<Steps>
+### Install the SDK
+
+**AptosKit** is available as a Swift package. To add it to your project,
+add the following to your `Package.swift` file:
+
+```swift filename="Package.swift"
+dependencies: [
+  .package(url: "https://github.com/mcxross/swift-aptos.git", .upToNextMajor(from: <version>))
+]
+```
+
+### Import the SDK
+
+Import the SDK in your Swift file:
+
+```swift filename="Main.swift"
+  import AptosKit
+```
+
+### Create the `ClientConfig` object
+
+This object is used to configure the client behavior. You can set `maxRetries`,
+`requestTimeout`, and `retryOnServerErrors` properties.
+
+```swift filename="Main.swift"
+  let config = ClientConfig(
+      followRedirects: true,
+      agent: "AptosClient",
+      likeAgent: nil,
+      requestTimeout: 5000,
+      retryOnServerErrors: 3,
+      maxRetries: 5,
+      cache: false,
+      proxy: nil
+  )
+```
+
+### Create the `AptosSettings` object
+
+This object is used to configure the Aptos network connection. You can set `network`,
+`fullnode`, and `faucet` properties.
+
+```swift filename="Main.swift"
+  let aptosSettings = AptosSettings(
+      network: .devnet,
+      fullNode: nil,
+      faucet: nil,
+      indexer: nil,
+      client: nil,
+      clientConfig: config,
+      fullNodeConfig: nil,
+      indexerConfig: nil,
+      faucetConfig: nil
+  )
```

Comment:
Fix indenting / formatting

Reply:
# Expansion

Unless there’s a specific required indentation for the documentation, please share it. However, this formatting follows Xcode’s style, which is generally considered the recommended approach:

![CleanShot 2024-07-23 at 11 12 10@2x](https://github.com/user-attachments/assets/598909e3-f3b0-47b3-8d41-e4bbc4d1c693)

## `pnpm fmt`
Not quite sure of the above command in the [docs](https://github.com/aptos-labs/developer-docs?tab=readme-ov-file#debugging), but it barely does anything.